src/stream/src/executor/aggregation/approx_count_distinct.rs

```diff
@@ -296,8 +357,38 @@ mod tests {
     use super::*;
 
     #[test]
-    fn test_insert_and_delete() {
-        let mut agg = StreamingApproxCountDistinct::new();
+    fn test_streaming_approx_count_distinct_insert_and_delete_dense() {
```

Comment:
seems the only difference from the `sparse` test case is the `DENSE_BITS`, is it possible to extract the common logic into a common function?